tao_compiler/mlir/disc/BUILD

```diff
@@ -967,6 +967,41 @@ cc_library(
     alwayslink = 1,
 )
 
+cc_library(
+    name = "disc_input_output_alias",
+    srcs = ["transforms/disc_input_output_alias.cc"],
+    hdrs = [
+        "transforms/passes.h",
+        "transforms/rewriters.h",
+    ],
+    deps = [
+        ":codegen_utils",
+        ":disc_util",
+        ":disc_ral",
+        ":fusion_utils",
+        ":lmhlo_disc",
+        ":pass_details",
+        ":placement_utils",
+        ":rng_uniform_custom_call",
+        ":topk_custom_call",
```

Comment:
please polish the dependencies, this target  do not need `qconv_custom_call` at least.

Reply:
OK～